warehouse/query-core/src/test/java/datawave/query/tracking/ActiveQueryLogTest.java

```diff
@@ -90,6 +95,33 @@ public void run() {
         }
     }
     
+    @Test
+    public void whenRetrievingInstance_givenNullName_shouldReturnsDefaultInstance() {
```

Comment:
ouch! How about a shorter name and some comments above it.

Reply:
I shortened the test case names a bit.